packages/web/src/features/languages/ManageLanguageView.tsx

```diff
@@ -73,59 +73,62 @@ export default function ManageLanguageView() {
           <span className="mx-2">-</span>
           <span>{language.data.code}</span>
         </ViewTitle>
-        <Form context={formContext} onSubmit={onSubmit} className="mb-8">
-          <div className="mb-2">
-            <FormLabel htmlFor="name">{t('name').toUpperCase()}</FormLabel>
-            <TextInput
-              id="name"
-              name="name"
-              className="w-full"
-              autoComplete="off"
-              defaultValue={language.data.name}
-              required
-              aria-describedby="name-error"
-            />
-            <InputError id="name-error" name="name" context="name" />
-          </div>
-          <div>
-            <Button type="submit">{t('update')}</Button>
-            <SubmittingIndicator className="ms-3" />
-          </div>
-        </Form>
-        <List>
-          <ListHeader>
-            <ListHeaderCell className="min-w-[120px]">
-              {t('name').toUpperCase()}
-            </ListHeaderCell>
-            <ListHeaderCell className="min-w-[120px]">
-              {t('email').toUpperCase()}
-            </ListHeaderCell>
-            <ListHeaderCell className="min-w-[120px]">
-              {t('roles').toUpperCase()}
-            </ListHeaderCell>
-            <ListHeaderCell />
-          </ListHeader>
-          <ListRowAction colSpan={4}>
-            <Link to="./invite">
-              <Icon icon="plus" className="me-1" />
-              {t('invite_user')}
-            </Link>
-          </ListRowAction>
-          <ListBody>
-            {members.data.map((member) => (
-              <ListRow key={member.userId}>
-                <ListCell header>{member.name}</ListCell>
-                <ListCell>{member.email}</ListCell>
-                <ListCell>
-                  {member.roles
-                    .map((role) => t('role', { context: role.toLowerCase() }))
-                    .join(', ')}
-                </ListCell>
-                <ListCell></ListCell>
-              </ListRow>
-            ))}
-          </ListBody>
-        </List>
+        <div className="flex flex-col gap-8">
```

Comment:
I usually do it because I don't like individually specifying spacing on each item when I can just specify it once in the parent. Less code changes. If you want to update spacing you just have to change it at one place